Tests/WalletConnectSharp.Sign.Test/SignTests.cs

```diff
@@ -384,50 +389,50 @@ public async void TestTwoUniqueSessionRequestResponse()
         }
         
         [Fact, Trait("Category", "integration")]
-        public async void TestTwoUniqueSessionRequestResponseUsingAddressProviderDefaults()
+        public async Task TestTwoUniqueSessionRequestResponseUsingAddressProviderDefaults()
```

Comment:
Is there a specific reason why most test methods return void instead of Task? If not, I recommend switching to Task to follow best practices.